index2.html

```diff
@@ -11,233 +11,98 @@
-				<div class="img-communication">
-					<img src="img/Vector2.svg" alt="communication">
-				</div>
-				<p class="text-container">
-					Communicate directly
-				</p>
-			</div>
-			<div class="box-book">
-				<div class="img-book">
-					<img src="img/Vector3.svg" alt="book-online">
-
-				</div>
-				<p class="text-container"></p>
-				Book online
-			</div>
-			<div class="box-reviews">
-				<div class="img-reviews">
-					<img src="img/Vector4.svg" alt="Real-guest-reviews">
-				</div>
-				<p class="text-container">
-					Real guest reviews
-				</p>
-			</div>
-		</div>
-	</div>
-	<!--//choice of places of rest-сontainer-->
-	<div class="container-info">
-		<div class="h1">
-			<h1 class="title-h1">Homes guests loves</h1>
-		</div>
-		<div class="content-box">
-			<div class="info-images">
-				<div class="images">
-					<div class="img">
-						<img src="img/Rectangle 48.png" alt="Hotel Leopold">
-					</div>
-					<div class="hotel-images">
-						<p class="text-info">Hotel Leopold</p>
-						<p class="text-infof">Saint Petersburg, Russia</p>
-					</div>
-				</div>
-			</div>
-			<div class="info-images">
-				<div class="images">
-					<div class="img">
-						<img src="img/Rectangle 49.png" alt="Apartment Sunshine">
-					</div>
-					<div class="hotel-images">
-						<p class="text-info">Apartment Sunshine</p>
-						<p class="text-infof">Santa Cruz de Tenerife, Spain</p>
-					</div>
-				</div>
-			</div>
-			<div class="info-images">
-				<div class="images">
-					<div class="img">
-						<img src="img/Rectangle 50.png" alt="Vysokie Tatry, Slowakia">
-					</div>
-					<div class="hotel-images">
-						<p class="text-info">Villa Kunerad</p>
-						<p class="text-infof">Vysokie Tatry, Slowakia</p>
-					</div>
-				</div>
-			</div>
-			<div class="info-images">
-				<div class="images">
-					<div class="img">
-						<img src="img/Rectangle 51.png" alt="Saint Petersburg, Russia">
-					</div>
-					<div class="hotel-images">
-						<p class="text-info">Hotel Leopold</p>
-						<p class="text-infof">Saint Petersburg, Russia</p>
-					</div>
-				</div>
-			</div>
-		</div>
+		<section class="section_search">
+			<h1>Discover stays
+				<br>
+				to live, work or just relax</h1>
+			<div class="section_label">
+				<label for="input_hotel" class="label">Your destination or hotel name</label>
+				<label for="input_date" class="label">Check-in — Check-out</label>
+			</div>
+			<form action="#" method="POST" class="form_input">
+				<input class="input-text" type="text" id="input_hotel" placeholder="New York">
+				<input class="input-data" type="date" id="input_date">
+				<input class="input_check" type="text" placeholder="2 Adults — 0 Children — 1 Room">
+				<input class="submit" type="submit" value="Search">
+			</form>
+			<div class="section_icon">
+				<a href="#"><img class="google-store-icon" src="LinkedSprite.svg#src-1"></a>
+				<a href="#"><img class="apple-icon" src="LinkedSprite.svg#src-5"></a>
+			</div>
```

Comment:
check height. Space shouldn't be clickable
![image](https://user-images.githubusercontent.com/20189313/128646566-89f60c3f-fdbd-4ce8-a3e0-9f82a0b72ad4.png)